jprov/jprovd/provider_commands.go

```diff
@@ -34,12 +34,12 @@ func StartServerCommand() *cobra.Command {
 	cmd.Flags().Int(types.FlagPort, 3333, "Port to host the server on.")
 	cmd.Flags().String(types.VersionFlag, "", "The value exposed by the version api to allow for custom deployments.")
 	cmd.Flags().Bool(types.HaltStraysFlag, false, "Debug flag to stop picking up strays.")
-	cmd.Flags().Uint16(types.FlagInterval, 32, "The interval in seconds for which to check proofs.")
+	cmd.Flags().Uint16(types.FlagInterval, 32, "The interval in seconds for which to check proofs. Must be >=120 if you need a custom interval")
 	cmd.Flags().Uint(types.FlagThreads, 10, "The amount of stray threads.")
 	cmd.Flags().Int(types.FlagMaxMisses, 16, "The amount of intervals a provider can miss their proofs before removing a file.")
-	cmd.Flags().Int64(types.FlagChunkSize, 10240, "The size of a single chunk.")
+	cmd.Flags().Int64(types.FlagChunkSize, 10240, "The size of a single file chunk.")
 	cmd.Flags().Int64(types.FlagStrayInterval, 20, "The interval in seconds to check for new strays.")
-	cmd.Flags().Int(types.FlagMessageSize, 500_000, "The max message size in bytes to submit to the chain at one time.")
+	cmd.Flags().Int(types.FlagAllMessagesSize, 500_000, "The max size of all messages in bytes to submit to the chain at one time.")
```

Comment:
We need to properly deprecate this by keeping both and ignoring `FlagAllMessageSize` if `FlagMessageSize` is set or else we will break some service files.

Reply:
Understood. Reverting back for now.